AuditEventProcessor.cpp

```diff
@@ -226,22 +285,21 @@ bool AuditEventProcessor::process_execve()
                 break;
             }
             case AUDIT_PATH: {
-                const char* field = auparse_find_field(_state, "item");
-                if (field == nullptr || strcmp(field, "0") != 0) {
+                if (path_seen) {
```

Comment:
While it seems like the PATH record for the exe always appears first, I don't think we should assume that is the case. We should only include the PATH record for which name==exe.